File: back-end/src/main/java/backend/model/po/Task.java
package backend.model.po;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import java.io.Serializable;
import java.util.Date;
import lombok.Data;

/**
 *
 * @TableName task
 */
@TableName(value ="task")
@Data
public class Task implements Serializable {
    /**
     *
     */
    @TableId(type = IdType.AUTO)
    private Integer taskId;

    /**
     *
     */
    private String name;

    /**
     *
     */
    private String detail;

    /**
     *
     */
    private Date startTime;

    /**
     *
     */
    private Date endTime;

    /**
     *
     */
    private Integer homeworkId;
    /**
     *
     */
    private Integer publish;

    @TableField(exist = false)
    private static final long serialVersionUID = 1L;
}

File: front-end/src/mock/task.js
import Mock from "mockjs";

const tasks = [
    {
        'taskId': 1,
        'name': '需求分析',
        'desc': '完成酒店管理系统的需求分析',
        'startTime': '2023-6-12',
        'endTime': '2023-6-14',
        'isPublish':1,
    },
    {
        'taskId': 2,
        'name': '界面设计',
        'desc': '完成酒店管理系统的界面设计',
        'startTime': '2023-6-15',
        'endTime': '2023-6-18',
        'isPublish':0
    },
]
Mock.mock(`/task/list`, 'get', (options) => {
    return{
        code:200,
        data:tasks
    }
})
